systemtest/src/main/java/io/strimzi/systemtest/templates/crd/KafkaTemplates.java

```diff
@@ -252,17 +252,42 @@ public static KafkaBuilder kafkaWithCruiseControl(String clusterName, int kafkaR
         return defaultKafka(kafka, clusterName, kafkaReplicas, zookeeperReplicas)
             .editSpec()
                 .editKafka()
-                    // faster cluster model generation tuning: reduce reporting and metadata refresh intervals
                     .addToConfig("cruise.control.metrics.reporter.metrics.reporting.interval.ms", 5_000)
                     .addToConfig("cruise.control.metrics.reporter.metadata.max.age.ms", 4_000)
                 .endKafka()
                 .editCruiseControl()
-                    // Extend active users tasks
+                    // the following configurations are set for better reliability and stability of CC related tests
+                    .addToConfig("max.active.user.tasks", 10)
```

Comment:
I can understand that metrics sampling helps the model generation to be fast but I don't get how setting `max.active.user.tasks` from 5 (default) to 10 can help. Do we have more tasks running in parallel on the CC? More than 5?

Reply:
You are right, that doesn't help, I just kept it to be aligned with the default CC configuration used in all the other tests. Not a big deal in the end.

Reply:
This is our common config for CC. When we were running tests in parallel, there were some issues with to low value for max.active.user.tasks. So in general it is our default value not related to these changes.